roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2

```diff
@@ -31,32 +31,32 @@ crd_paths:
 {% endif %}
 ocp_versions: "{{ ocp_versions }}"
 {% if operator_feature_disconnected is defined %}
-operator_feature_disconnected: {{ operator_feature_disconnected }}
+operator_feature_disconnected: "{{ operator_feature_disconnected | lower }}"
```

Comment:
can try to use just quotes without `lower`
```
- operator_feature_disconnected: {{ operator_feature_disconnected }}
+ operator_feature_disconnected: "{{ operator_feature_disconnected }}"
```

Otherwise user can pass value `MAYBE` and it would be converted to `maybe`
and that is not really expected

Reply:
without `lower` Ansible parses those values as "True"/"False", this goes back to the discussion in the first thread

Reply:
But somehow if I run a real OVP job then in parsed_operator_data.yml the values are "true"/"false", so maybe I should keep it as is here? 
http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-redhat-operator-bundle-image-validation-test/e2e-test-operator-bundle-container-8.0-11/3e4859f7-f8dd-4db9-978d-6fe7176af6f3/parsed_operator_data.yml

However, now I see a different issue with using quotes. What if user passes a boolean true, and we convert it to "true" by using the quotes around it which makes it a string

Reply:
> What if user passes a boolean true, and we convert it to "true" by using the quotes around it which makes it a string

That is tiny disadvantage and should be probably discussed with OCP folks; whether it should fail or not.
But IIUC operator sdk should validate that in the future anyway; and thus CVP would fail a bit later than in parsing phase.

unfortunately, we cannot avoid conversion of bool -> string with current implementation of parse metadata in `ansible`
But IMHO it is a bit smaller problem than automatic lowercasing of strings

BTW one can see that in unit test that bool is converted to string as
`operator_feature_token_auth_azure: "False"\noperator_feature_token_auth_gcp: "False"\n'`
which is not lowercase . So it can be easily detected even though you would not be able to distinguish between
```
features.operators.openshift.io/disconnected: true
vs
features.operators.openshift.io/disconnected: "True"
```

Reply:
Yes, I agree that using `lower` is definitely not a good idea. In this case, I'll return to my first implementation where I have to use "True"/"False" in the test. Seems like it is the best option.
Thank you!

Reply:
`"True"/"False"` 
That would be for boolean. Which is OK for negative test case.
It should be just clear from test either by adding comments or jsut more self descriptive test name
(or combination of both :-)